lib/mongoid/errors/invalid_around_callback.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+# rubocop:todo all
```

Comment:
The `rubocop:todo all` line is a bandaid for existing files, so that we can fix linter errors on a file-by-file basis as we work in them. For new files, let's leave that line out and just make them linter-approved from the start. Note that a lot of our existing files have linter issues, so you'll almost certainly see the linter complain about this file, at first. Fortunately, it's a small file. :)

(You can run the linter locally on a single file, with `rubocop path/to/file`, or on the whole project with just `rubocop`.)

```suggestion
```

Reply:
Thank you for the context. Pushed this change because the file is linter-approved now!